Ginger/GingerCoreNET/WorkSpaceLib/Solution.cs

```diff
@@ -440,8 +442,37 @@ public object CreateNewReportTemplate()
         }
 
 
+        private ObservableList<VariableBase> mVariables;
+        /// <summary>
+        /// Been used to identify if Solution Variables were lazy loaded already or not
+        /// </summary>
+        public bool VariablesLazyLoad { get { return (mVariables != null) ? mVariables.LazyLoad : false; } }
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5848147)